insights/parsers/ssh_client_config.py

```diff
@@ -124,6 +127,14 @@ class EtcSshConfig(SshClientConfig):
     pass
 
 
+@parser(Specs.ssh_config_d)
+class EtcSshConfigD(SshClientConfig):
+    """
+    This Parser reads the files ``/etc/ssh/ssh_config.d/``
```

Comment:
Missing the trailing `*` on the glob.

Reply:
Please add the docstrings for this parser consistent with the [documentation guidelines](https://insights-core.readthedocs.io/en/latest/docs_guidelines.html).  I also noticed that the base class attributes are not documented so please either include those in the base class docstring or in the docstring for your class.  

Please also add testing for this parser.  The base class could change and break this parser.